mise.toml

```
@@ -6,8 +6,15 @@
 #     "./tests/mise.tcp.toml",
 #     "./tests/mise.tls.toml",
 # ]
+
+[tools]
+rust = "latest"
+"cargo:cargo-binstall" = "latest"
+"cargo:sqlx-cli" = "latest"
+
+
 [task_config]
-includes = ["tasks", "tasks/postgres.toml", "tasks/rust.toml"]
+includes = ["tasks", "tasks/postgres.toml"]
```

chore: line per task, to make cleaner diffs

```suggestion
includes = [
  "tasks", 
  "tasks/postgres.toml"
]
```